README.md

```diff
@@ -0,0 +1,111 @@
+Substrait Consumer Integration Tests
+====================================
+
+Table of Contents
+=================
+* [Overview](#Overview)
+* [Setup](#Setup)
+* [How to Run Tests](#How-to-Run-Tests)
+* [Test Files](#Test-Files)
+  * [Test Case Args](#Test-Case-Args)
+  * [Substrait Plans](#Substrait-Plans)
+  * [SQL Queries](#SQL-Queries)
+* [How to Add New Consumers](#How-to-Add-New-Consumers)
+
+# Overview
+This testing repository provides instructions on how to add and run substrait consumer 
+integration tests.  These tests take in parquet files, a SQL query, and a json-formatted substrait 
+query plan as input.  The tests will run the SQL query on DuckDB, and the substrait plan on the 
+specified consumer, and then compare the two results. The test data is created using DuckDB at the 
+start of the test class using the `prepare_tpch_parquet_data` fixture, which is located in 
+`tests/conftest.py`.
+
+# Setup
+
+Create and activate your conda environment with python3.9:
+```commandline
+conda create -y -n substrait_consumer_testing -c conda-forge python=3.9
+conda activate substrait_consumer_testing
+```
+
+Install requirements from `tests` directory:
+```commandline
+pip install -r requirements.txt
+```
```

Comment:
```suggestion
Install requirements:
```commandline
pip install -r tests/requirements.txt
```
```

Or maybe you could phrase it "Install requirements (must run from the `tests` directory)

Reply:
I actually moved the requirements.txt to the top level directory.  I'll update the README.